base/doc/fntguide.tex

```diff
@@ -1874,6 +1901,66 @@ \subsection{The order of declaration}
     declarations.
 \end{itemize}
 
+\subsection{Font series defaults per document family}
+
+\NEWfeature{2020/02/02}
+With additional weights and widths being available in many font families
+nowadays, it is more likely that somebody will want to match, say, a
+medium weight serif family with a semi-light sans serif family, or that
+with one family one wants to use the bold-extended face when |\textbf|
+is used, while with another it should be bold (not extended) or
+semi-bold, etc.  \LaTeX{} stores family specific defaults for series in
+these internal macros:
+\begin{decl}
+  |\mdseries@rm| \\
+  |\mdseries@sf| \\
+  |\mdseries@tt| \\
+  |\bfseries@rm| \\
+  |\bfseries@sf| \\
+  |\bfseries@tt|
+\end{decl}
+
+The default values can be overwritten by the author level macro in
```

Comment:
replace auther level macro with declation

maybe

the default values can be altered using a \cs{DeclareFontSeriesDefault} declaration